content/blog/hiring-2020-web-developer.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "We're hiring: Full-stack web developer"
+date: 2020-04-06T11:50:00+01:00
+author: RESIDE
+slug: "hiring-2020-malaria-developer"
+tags:
+ - hiring
+---
+
+[We are hiring!](https://www.imperial.ac.uk/jobs/description/MED01818/research-software-engineer-web-development/)  While many people of the department and some of the RSE group are focussed on COVID-19 at the moment, research and public health action on  malaria, HIV, and other diseases continues in the department and we're looking for someone to help us develop web applications that translate research into public health outcomes.
```

Comment:
This could be phrased to make it sound more exciting, maybe throw in the fact that malaria and HIV continue to have a higher global burden so this is high impact work

Reply:
great point - have added a bit more here